src/photos/ducks/albums/components/AlbumsView.spec.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import PropTypes from 'prop-types'
+import AppLike from '../../../../../test/components/AppLike'
+import { mockedRouter } from '../../../../../test/__mocks__/mockedRouter'
+
+import AlbumsView from './AlbumsView'
+import { createMockClient } from 'cozy-client'
+
+describe('Albumsview', () => {
+  const client = createMockClient({})
+  let component
+  const setup = data => {
+    component = mount(
```

Comment:
Same as in last commit, setup could return `component` instead of relying on global variable.